source/php/Component/Acceptance/Acceptance.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace ComponentLibrary\Component\Acceptance;
+
+class Acceptance extends \ComponentLibrary\Component\BaseController
+{
+    public function init()
+    {
+     //Extract array for eazy access (fetch only)
+        extract($this->data);
+
+        $this->data['labels'] = $labels;     
+
+        if(isset($modifier)) {
+            $this->data['classList'][] = $this->getBaseClass() . $modifier;
+            
+            if($modifier == '--video') {
+                $this->data['isVideo'] = true;
+            }
+            else {
+                $this->data['isVideo'] = false;
+            }
+            $this->data['modifier'] = $modifier;
+        } else {
+            $this->data['modifier'] = '--none';
+        }
+
+        if(isset($height)) {
+            $this->data['attributeList']['style'] = "height:" . $height . "px;"; 
```

Comment:
I would use either `'height:' . $height . 'px;'` or `"height:{$height}px;";`

Reply:
Reason for this is to not have empty attributes on elements. This will be prevented using attributeList :)